moveit_ros/move_group/include/moveit/move_group/capability_names.h

```diff
@@ -41,6 +41,25 @@
 
 namespace move_group
 {
+// These capabilities are loaded unless listed in disable_capabilities
+// clang-format off
+static const char* DEFAULT_CAPABILITIES[] = {
```

Comment:
is char* really the best option here? MoveIt! almost always uses ``std::string`` - and this structure gets inserted into an ``std::string`` set